README.md

```diff
@@ -46,6 +51,45 @@ pip to install directly from the master branch on github:
 pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
 ```
 
+## Using KerasCV with Keras Core
+
+As of version 0.6.0, KerasCV supports multiple backends with Keras Core out of 
+the box. There are three ways to configure KerasCV to run with multi-backend 
+support:
+
+1. Via the `KERAS_BACKEND` environment variable. If set, then KerasCV will be 
+using Keras Core with the backend specified (e.g., `KERAS_BACKEND=jax`).
+2. Via the environment variable `KERAS_CV_MULTI_BACKEND` (e.g, 
+`KERAS_CV_MULTI_BACKEND=true`).
+3. Via the `.keras/keras.json` and `.keras/keras_cv.json` config files (which 
+are automatically created the first time you import KerasCV).
+- Set your backend of choice in `.keras/keras.json`; e.g., `"backend": "jax"`. 
+- Set `"multi_backend": True` in `.keras/keras_cv.json`.
+
+Once that configuration step is done, you can just import KerasCV and start 
+using it on top of your backend of choice:
+
+```python
+import keras_cv
+import keras_core as keras
+
+filepath = keras_core.utils.get_file(origin="https://i.imgur.com/gCNcJJI.jpg")
+image = np.array(keras.utils.load_img(filepath))
+image_resized = ops.image.resize(image, (640, 640))[None, ...]
+
+model = keras_cv.models.YOLOV8Detector.from_preset(
+    "yolo_v8_m_pascalvoc",
+    bounding_box_format="xywh",
+)
+predictions = model.predict(image_resized)
+```
+
+Until Keras Core is officially released as the new Keras, KerasCV will use 
```

Comment:
I might focus on the `KERAS_BACKEND` env variable approach instead of the json files, as IMO it's a little more practical for quick changes, and way easier for use in colab.

Reply:
Let me know if the current wording makes sense

Reply:
Looks great!

"the new Keras" -> **Keras 3.0**

In keeping with the announcement copy?